db/data/20240918210326_create_and_index_default_sandboxes.rb

```
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateAndIndexDefaultSandboxes < ActiveRecord::Migration[7.1]
+  def up
+    Jurisdiction.find_each(&:save)
+    Jurisdiction.reindex
+  end
+
+  def down
+    raise ActiveRecord::IrreversibleMigration
```

probably don't need to actually raise an error here